development/extensions/skeleton_extension.rst

```diff
@@ -30,17 +30,17 @@ installed into a phpBB board just the same as any other.
 
 .. note::
 
-    The Skeleton Extension is a tool for extension developers. As such,
-    it is intended to be installed in a phpBB board used for
-    offline/local development of extensions. It is not necessarily
-    intended to be installed on live/production web sites.
+    The Skeleton Extension is intended to be installed in a phpBB
+    board used for offline/local development of extensions. It is
+    not necessarily intended to be installed on live/production web
```

Comment:
Since it can be, should be better to add some appropriate perms to this extension, my thought. I am aware an issue has been rejected which relates to this one of mine comment, but I would like to stress this argument. How would you think to deny the instalation on a live board?